src/pulselistener/pulselistener/listener.py

```diff
@@ -56,6 +65,18 @@ def __init__(self, configuration):
         assert len(diffs) == 1
         self.latest_id = diffs[0]['id']
 
+        # Add web route for new code review
+        if self.mode == MODE_PHABRICATOR_WEBHOOK:
+            self.routes.append(web.post('/codereview/new', self.new_code_review))
+
+        # Load secure projects
```

Comment:
Can you explain this?

Reply:
Are there projects that are completely consisting of secure revisions?

Reply:
Ah, I see https://bugzilla.mozilla.org/show_bug.cgi?id=1539745#c7 now. Let's wait on that.

Reply:
Current logic is OK. It just need a retry loop + try/catch around it